src/DependencyInjection/JaegerExtension.php

```diff
@@ -18,13 +18,40 @@ public function load(array $configs, ContainerBuilder $container)
             new FileLocator(__DIR__ . '/../Resources')
         );
         $loader->load('services.yml');
-
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
-
         if ($this->isConfigEnabled($container, $config['denylist'])) {
             $container->setParameter('jaeger.sampler.operation_denylist', $config['denylist']['operation_names']);
             $loader->load('denylist.yml');
         }
+        if ($this->isConfigEnabled($container, $config['name_generator'])) {
+            $container->setParameter('jaeger.name.max_length', (int)$config['name_generator']['max_length']);
```

Comment:
Casting can be dropped as we changed config node type to `IntegerNode`

```suggestion
            $container->setParameter('jaeger.name.max_length', $config['name_generator']['max_length']);
```